hook.py

```diff
@@ -16,25 +17,53 @@
 sphinx_docs_root = plugin_root / 'sphinx-docs'
 html_docs_root = sphinx_docs_root / '_build' / 'html'
 
+logger = logging.getLogger("fieldmanual")
+
 
 async def landing(_):
     return web.FileResponse(str(plugin_root / 'static' / 'opener.html'))
 
 
-async def build_docs():
-    process = await asyncio.create_subprocess_exec(
-        'sphinx-build', '%s/' % (sphinx_docs_root,), str(html_docs_root), '-b', 'html', '-c', str(sphinx_docs_root),
-        stdout=asyncio.subprocess.PIPE, stderr=asyncio.subprocess.PIPE)
-    stdout, stderr = await process.communicate()
-    logging.debug(stdout)
-    if process.returncode:
-        logging.error('doc generation failed: %s' % (stderr,))
+def _run_sphinx_build():
+    import sys
+    import io
+
+    import sphinx.cmd.build
+
+    out = io.StringIO()
+    err = io.StringIO()
+    sys.stdout = out
```

Comment:
not sure if there was a cleaner way to do this, but this seems to work.  stdout / stderr are only rerouted for the child process.